cmd/rqlite/main.go

```diff
@@ -156,9 +158,24 @@ func expvar(ctx *cli.Context, cmd, line string, argv *argT) error {
 
 func sendRequest(ctx *cli.Context, makeNewRequest func(string) (*http.Request, error), urlStr string, argv *argT) (*[]byte, error) {
 	url := urlStr
+	rootCAs := (*x509.CertPool)(nil)
```

Comment:
Why not just do `var rootCAs *x509.CertPool`? That seems more natural.

Reply:
Yes, that's better. Updated.